python-regression/util/milestone_logic/milestones.py

```diff
@@ -38,3 +38,12 @@ def issue_milestone(address, api, index, *reference_transaction):
     api.broadcast_and_store(milestone['trytes'])
 
     return milestone
+
+
+def update_latest_milestone(config, node, milestone):
+    if 'latestMilestone' not in config:
+        config['latestMilestone'] = {}
+
+    milestone_hash = Transaction.from_tryte_string(milestone['trytes'][0]).hash
+    milestone_hash2 = Transaction.from_tryte_string(milestone['trytes'][1]).hash
+    config['latestMilestone'][node] = [milestone_hash, milestone_hash2]
```

Comment:
Question:
I am just a bit confused on what you are doing here.
Is this a global variable?
Shouldn't it be in `world` and not `config`?

If I am correct please change.
If I am misunderstanding something please enlighten me :-)

Reply:
Yes it is storing it in the world global. The config value is the world.config that's been passed to it from higher in the code. This updates the latest milestone in the world environment.